wgengine/netstack/netstack.go

```diff
@@ -866,6 +826,53 @@ func (ns *Impl) inject() {
 	}
 }
 
+func (ns *Impl) shouldSendToHost(pkt *stack.PacketBuffer) bool {
+	// Determine if the packet is from a service IP, in which case it
```

Comment:
Nit: for me reading this first time it was not immediately clear what 'service' is in this context

Reply:
Thanks; good point. Clarified with a comment.